nexus/pkg/server/sender.go

```diff
@@ -237,6 +256,131 @@ func (s *Sender) getValueConfig(config map[string]interface{}) map[string]map[st
 	return datas
 }
 
+func (s *Sender) sendResume(run *service.RunRecord) error {
```

Comment:
nit:   this doesnt follow normal conventions of sendNoun where Noun is a record type.  can we rename it?

Reply:
Oh yeah was planning to do that. Thanks